source/apiVolontaria/volunteer/tests/tests_view_Participations.py

```diff
@@ -147,7 +143,6 @@ def test_create_new_participation(self):
         content = json.loads(response.content)
 
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
-        self.assertEqual(content['subscription_date'], subscription_date_str)
```

Comment:
I have removed this check, I was receiving something like this for the content :  
`{'id': None, 'event': 3, 'user': {'id': 4, 'username': 'John2', 'email': 'john2@volontaria.com', 'first_name': 'Lisa', 'last_name': 'Sawyer', 'is_active': True, 'is_superuser': False, 'phone': None, 'mobile': None, 'managed_cell': []}, 'standby': False, 'subscription_date': None, 'presence_duration_minutes': None, 'presence_status': 'I'}`

Reply:
Yeah but it's not good to receive `None` in place of the good value...
Why did this change ?